src/init/features/project.ts

```diff
@@ -172,21 +95,22 @@ export async function doSetup(setup: any, config: Config, options: any): Promise
     choices,
   });
 
-  let projectInfo;
+  let projectMetaData;
   if (projectSetupOption === OPTION_USE_PROJECT) {
-    projectInfo = await getProjectInfo(options);
+    projectMetaData = await getOrPromptDesiredProject(options);
   } else if (projectSetupOption === OPTION_NEW_PROJECT) {
-    projectInfo = await promptAndCreateNewProject();
+    projectMetaData = await promptAndCreateNewProject();
   } else {
-    // Do nothing if use choose NO_PROJECT
+    // Do nothing if user chooses NO_PROJECT
     return;
   }
 
-  utils.logBullet(`Using project ${projectInfo.label}`);
+  const { id, label, instance, location } = toProjectInfo(projectMetaData);
```

Comment:
nit: Rather than having all these separate `const`s, I prefer how it was before, where they were all grouped inside a `projectInfo` object.

eg. The `id` field on its own is not particularly clear, but `projectInfo.id` is more clear.

Reply:
Reverted